lib/request.ts

```diff
@@ -328,7 +388,6 @@ export function getRequest({
 	 * 		throw error
 	 * )
 	 */
-	exports.interceptors = interceptors;
```

Comment:
All the docs for `inteceptors` are directly above this, I think you'll need to move the docs somewhere else now for it to all format/generate correctly

Reply:
Had some trouble to get this right but eventually I had to take a shortcut. Hope you are fine with it :)
Fun fact, the doc generation was already broken on master...